Firmware/ultralcd.cpp

```diff
@@ -772,21 +771,6 @@ void lcdui_print_status_screen(void)
 // Main status screen. It's up to the implementation specific part to show what is needed. As this is very display dependent
 void lcd_status_screen()                          // NOT static due to using inside "Marlin_main" module ("manage_inactivity()")
 {
-	if (firstrun == 1) 
```

Comment:
I'm not completely convinced that the removal of this code is correct. Please prepare a detailed feature test for this scenario.

Reply:
I'll do a factory reset later and look at the statistics. i already tested the welcome message change, so that is fine, but haven't tested the statistics init yet. Even though that should be fine, I'll test it anyway a bit later